requirements/base.txt

```diff
@@ -13,4 +13,6 @@ ordered-set==2.0.1                  # MIT
 Markdown==2.6.6    					# BSD
 
 -e git+https://github.com/edx/opaque-keys.git@d45d0bd8d64c69531be69178b9505b5d38806ce0#egg=opaque-keys
+# custom opaque-key implementations for ccx
+git+https://github.com/jazkarta/ccx-keys.git@e6b03704b1bb97c1d2f31301ecb4e3a687c536ea#egg=ccx-keys
```

Comment:
Hmm... this repo is titled "obsolete-ccx-keys" now. Any reason we can't use the normal edx-ccx-keys? https://pypi.python.org/pypi/edx-ccx-keys/0.2.1